hexrd/ui/indexing/fit_grains_options_dialog.py

```diff
@@ -71,13 +71,11 @@ def all_widgets(self):
     def show(self):
         self.ui.show()
 
-    @Slot()
```

Comment:
Why remove these?

Reply:
They aren't necessary. If there is a benefit in having them, let me know.

Reply:
When you have a slot parameter, don't they help enforce the type?

Reply:
Yeah, it looks like it. I guess we can use them if we want to enforce a type.